README.md

```diff
@@ -445,6 +445,16 @@ dockerBuild {
  * **git**: Adds labels derived from git: current branch, origin repository, latest commit hash and author.
    This label contributor is enabled by default and can be disabled by calling `fromGit(false)`
 
+#### fromProject with variable name
```

Comment:
Why are you putting the documentation here?
I linked the section where it needs to be inserted, it's under the "Plugin `eu.xenit.docker-compose`: Inject built docker images into docker-compose" title.

Reply:
When I click the link, it sent me to this part of the docs. It's weird though cause the name in the link says something different. Will adjust.